corehq/apps/cleanup/management/commands/generate_form_case_consistency_list.py

```diff
@@ -116,7 +119,7 @@ def _case_ids_in_es(submission):
     def handle(self, *args, **options):
         self.es = get_es()
         try:
-            from_date = datetime.strptime(options['from_date'], "%Y-%m-%d")
```

Comment:
This was returning a datetime, but it was being passed to a function that just used the date portion of it again—anyway I changed it to return a date since that doesn't break anything and is cleaner